config/dev.exs

```diff
@@ -39,4 +39,5 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :jessica_boss, JessicaBoss.Repo,
   adapter: Sqlite.Ecto2,
-  database: "jessica_boss_dev.sqlite3"
+  database: "jessica_boss_dev.sqlite3",
```

Comment:
It would be nice to have a proper folder (e.g., `/db`, `/tmp`) for these database files, instead of leaving them on the root folder.